### ISSUE TITLE
Is there a way to return the app the user selects when using showLocation?

### ISSUE THREAD

**Issue by acoulon99** [tschoffelen/react-native-map-link#116]
Would something like this be ok here?
https://github.com/tschoffelen/react-native-map-link/blob/1f2af73220b6103d43dbb700463320d068c53c2d/src/index.js#L163

```
return Linking.openURL(url).then(() => Promise.resolve(app))
```

**Comment by tschoffelen** [comment]
Yes, that could work!

**Comment by tschoffelen** [comment]
Fixed in #117.
